src/scss/base.scss

```diff
@@ -33,9 +33,27 @@ body {
 }
 
 .content {
-  display: flex;
-  flex-direction: column;
-  height: 100%;
+  margin-left: $sidebar-width;
+  max-width: 100%;
   overflow: auto;
-  width: 100%;
+  width: 1000px;
+}
+
+.title {
+  font-size: 32px;
+  font-weight: 700;
+  margin-top: 14px;
+  padding: 14px;
+}
+
+@media screen and (min-width: 1600px) {
```

Comment:
why this media query?

Reply:
Because on big screens he show it this way:
<img width="808" alt="screen shot 2019-01-03 at 11 39 21" src="https://user-images.githubusercontent.com/28515670/50649672-87b18500-0f4c-11e9-9632-15d34ea7f0a0.png">
 and with this media query change for this:
<img width="817" alt="screen shot 2019-01-03 at 11 39 30" src="https://user-images.githubusercontent.com/28515670/50649702-9ac45500-0f4c-11e9-8e8d-01300b6754a9.png">

Reply:
But if you set the margins in the element, there's no need to do it in the media query :smile: